releases/release-5.0.4.md

```diff
@@ -0,0 +1,169 @@
+        - Fix gRPC keepalive error when memory pressure is high. [#2719](https://github.com/pingcap/ticdc/pull/2719)
+        - Optimize memory management when unified sorter is using memory to sort. [#2711](https://github.com/pingcap/ticdc/pull/2711)
+        - Fix a bug that causes TiCDC to panic on an unsigned tinyint [#2655](https://github.com/pingcap/ticdc/pull/2655)
+        - Fix open protocol, don't output an empty value when there is no change in one transaction. [#2620](https://github.com/pingcap/ticdc/pull/2620)
+        - Fixed a bug in DDL handling when the owner restarts. [#2610](https://github.com/pingcap/ticdc/pull/2610)
+        - This PR make the old owner handle DDL in Async mode. [#2605](https://github.com/pingcap/ticdc/pull/2605)
+        - Prohibit operating TiCDC clusters across major and minor versions [#2598](https://github.com/pingcap/ticdc/pull/2598)
+        - Fix a bug in metadata management [#2558](https://github.com/pingcap/ticdc/pull/2558)
+        - Add a global gRPC connection pool and share gRPC connections among kv clients. [#2533](https://github.com/pingcap/ticdc/pull/2533)
+        - Fix a bug that multiple processors could write the same table when this table is re-scheduling [#2492](https://github.com/pingcap/ticdc/pull/2492)
+        - Optimize workerpool for fewer goroutines when concurrency is high. [#2487](https://github.com/pingcap/ticdc/pull/2487)
+        - fix outdated capture info may appear in capture list command [#2466](https://github.com/pingcap/ticdc/pull/2466)
+        - Fix a bug that owner could meet ErrSchemaStorageTableMiss error and reset a changefeed by accident. [#2458](https://github.com/pingcap/ticdc/pull/2458)
+        - fix the bug that changefeed cannot be removed if meet GcTTL Exceeded Error [#2456](https://github.com/pingcap/ticdc/pull/2456)
+        - fix a bug where synchronizing large tables to cdclog failed. [#2445](https://github.com/pingcap/ticdc/pull/2445)
+        - Fix CLI back-compatibility [#2413](https://github.com/pingcap/ticdc/pull/2413)
+        - Reduce goroutine usage when a table's region transfer away from a TiKV node [#2377](https://github.com/pingcap/ticdc/pull/2377)
+        - Remove file sorter. [#2326](https://github.com/pingcap/ticdc/pull/2326)
+        - puller,mounter,processor: always pull the old value internally [#2305](https://github.com/pingcap/ticdc/pull/2305)
+        - Fix minor runtime panic risk [#2299](https://github.com/pingcap/ticdc/pull/2299)
+        - Fix potential DDL loss when owner crashes while executing DDL [#2292](https://github.com/pingcap/ticdc/pull/2292)
+        - Don't resolve lock immediately after a region is initialized. [#2265](https://github.com/pingcap/ticdc/pull/2265)
+        - Fix extra partition dispatching when adding new table partition. [#2263](https://github.com/pingcap/ticdc/pull/2263)
+        - Better err msg when PD endpoint missing certificate [#2185](https://github.com/pingcap/ticdc/pull/2185)
```

Comment:
move `Better err msg when PD endpoint missing certificate [#2185](https://github.com/pingcap/ticdc/pull/2185)` to Improvements